subsys/bluetooth/shell/bt.c

```diff
@@ -192,11 +201,68 @@ static bool data_cb(struct bt_data *data, void *user_data)
 	}
 }
 
+static void print_data_set(uint8_t type, uint8_t set_value_len,
+						   const uint8_t *scan_data, uint8_t scan_data_len)
```

Comment:
Some indentation is off here

Reply:
Weird, looks fine here - with tabs.

Reply:
Zephyr has byzantine rules for indentation and alignment:

 - 8 spaces indentation
 - align insides of parenthesis split over multiple lines so they start on the column after '('
 - then greedily replace any n*8 spaces at the start of the line with n tab characters. That applies regardless if the spaces are for indentation or alignment.

Your editor probably has tab-stop set to 4, so it looks right to you even if it's not following the rules.

You should just run clang-format and have it take care of it.

Reply:
Yes, surprising indeed. It's been a long time since my last contribution, but now I recall those.... guidelines. Fixed my workspace setup.